integration_tests/pages/page.ts

```diff
@@ -14,6 +14,7 @@ export default abstract class Page {
   constructor(private readonly title: string) {
     this.checkOnPage()
     cy.injectAxe()
+    cy.configureAxe({ rules: [{ id: 'aria-allowed-attr', reviewOnFail: true }] })
```

Comment:
I wonder if we can keep these rules in a JSON file in the root of the project, so they're easier to refer to?

Reply:
I'm hoping this rule will be temporary and I feel like adding a seperate config file for the rules would encourage turning off rules in future rather than fixing the issues they highlight.

Reply:
Yeah that makes sense